README.md

```diff
@@ -2411,6 +2416,8 @@ IEEE Trans. Pattern Anal. Mach. Intell., 45(1)785:795, 2022. [Full PDF incl. Sup
 - <a name="Wang22icra"></a>Wang, Z., Yuan, D., Ng, Y., Mahony, R.,   
 *[A Linear Comb Filter for Event Flicker Removal](https://doi.org/10.1109/ICRA46639.2022.9812003)*,  
 IEEE Int. Conf. Robotics and Automation (ICRA), 2022. [PDF](https://arxiv.org/pdf/2205.08090.pdf), [Code and Data](https://github.com/ziweiWWANG/EFR), [Youtube](https://www.youtube.com/watch?v=1XgOknDIV00).
+- [Guillen-Garcia et al., IEEE Access, 2022](#Guillen-Garcia22).
+*M-out-of-N Event Declaration Filter for Improved Event Detection (Unsupervised adaptive multi-object tracking-by-clustering algorithm with a bio-inspired system)*.
```

Comment:
also in two separate lines, please

Reply:
almost there.

Reply:
Do I need to make any new changes? :)

Reply:
you need to add two blank spaces at the end of the line to force a line break. Please check the preview when creating a PR or a commit.